privateca/privateca_test.go

```diff
@@ -236,4 +236,20 @@ func TestCreateCa(t *testing.T) {
 			t.Errorf("disableCa got %q, want %q", got, expectedResult)
 		}
 	})
+
+	t.Run("listCas", func(t *testing.T) {
```

Comment:
issue: please move this into its own test case and remove the call to `t.Run()`.

We are in the process of updating the Samples Style Guide with this guidance:
https://github.com/GoogleCloudPlatform/samples-style-guide/pull/122

Reply:
Done, although I don't see a big benefit of doing it. In case of CA tests I had them all grouped in [single test](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/privateca/privateca_test.go#L164) with multiple t.Runs, which has following benefits:
 - we don't have to create a new CAPool for each individual test (it gets created only once for all t.Runs)
 - we still can see what exactly failed in the logs (go test -v) and summaries

One of the next tasks are certificate samples, which require a CAPool and a CA. Splitting them into individual tests will make result in creating a new CAPool and CA in each of the certificate tests.

Reply:
I appreciate your flexibility. There are several benefits to having 1-to-1 mapping of test cases to snippets:

- We can isolate a single test case using `go test --run TestCaseName`
- We can skip individual test cases that are indefinitely flaky
- We can ensure that teardown occurs on a test-by-test basis AND isolate tests that fail to tear down

This is also the standard that all of the other languages follow.